src/store/writer.rs

```diff
@@ -65,7 +65,7 @@ impl StoreWriter {
         VInt(doc_num_bytes as u64).serialize(&mut self.current_block)?;
         self.current_block.write_all(serialized_document)?;
         self.doc += 1;
-        if self.current_block.len() > BLOCK_SIZE {
```

Comment:
that's true for the other stores, and it does not matter much, but shouldn't we make that `>=`?.

This is almost philosophy at this point, but my mental model is 
"We close a block once it is full". "Being full means being greater or equal to BLOCK_SIZE"

Reply:
I just went with what was already there, personally I don't think it makes much difference since blocks are allowed to overflow anyway;
Happy to go either way on this one